src/Optimization/hiopKKTLinSysSparse.cpp

```diff
@@ -144,22 +147,22 @@ namespace hiop
       Msys->setSubDiagonalTo(nx, neq, -delta_cc, dest_nnz_st); dest_nnz_st += neq;
 
       /* we've just done above the (1,1) and (2,2) blocks of
-      *
-      * [ Hx+Dxd+delta_wx*I           Jcd^T          Jdd^T   ]
-      * [  Jcd                       -delta_cc*I     0       ]
-      * [  Jdd                        0              M_{33} ]
-      *
-      * where
-      * M_{33} = - (Dd+delta_wd)*I^{-1} - delta_cd*I = - Dd_inv - delta_cd*I is performed below
-      */
+       *
```

Comment:
only indentation changed, please rollback

Reply:
This actually fixed indentation that was off by one space.